### PR TITLE
NOISSUE: allow lr to react on pulse

### PR DESCRIPTION
**Pull request by Vany**
** touch lr.onPulse from pm**

### CODE REVIEW

File: logicrunner/logicrunner_test.go [hunk 1]
```diff
@@ -82,12 +81,19 @@ func PrepareLrAmCb(t testing.TB) (core.LogicRunner, core.ArtifactManager, *testu
...
 		MessageBus: &testMessageBus{LogicRunner: lr},
 	}), "starting logicrunner")
-	lr.OnPulse(*pulsar.NewPulse(configuration.NewPulsar().NumberDelta, 0, &pulsar.StandardEntropyGenerator{}))
-
+	log.Warn("SETTING PULSE")
+	err = l.GetPulseManager().Set(*pulsar.NewPulse(configuration.NewPulsar().NumberDelta, 100, &pulsar.StandardEntropyGenerator{}))
+	assert.NoError(t, err, "pulse set")
+	if err != nil {
+		t.Fatal("pulse set died, ", err)
+		panic(42)
```
Comment (aalekseev-ru) [inline logicrunner/logicrunner_test.go]: do we really need panic here?

File: logicrunner/logicrunner_test.go [hunk 2]
```diff
@@ -102,7 +108,7 @@ func PrepareLrAmCb(t testing.TB) (core.LogicRunner, core.ArtifactManager, *testu
 func ValidateAllResults(t testing.TB, lr core.LogicRunner) {
 	rlr := lr.(*LogicRunner)
 	for ref, cr := range rlr.caseBind.Records {
-		assert.Equal(t, configuration.NewPulsar().NumberDelta, uint32(rlr.caseBind.Pulse.PulseNumber), "right pulsenumber")
+		//assert.Equal(t, configuration.NewPulsar().NumberDelta, uint32(rlr.caseBind.Pulse.PulseNumber), "right pulsenumber")
```
Comment (aalekseev-ru) [inline logicrunner/logicrunner_test.go]: it's a copy of uncommented string

File: logicrunner/logicrunner_test.go (hunk 1 above)
Comment (Vany) [reply logicrunner/logicrunner_test.go]: Of course no, thanx

File: logicrunner/logicrunner_test.go (hunk 2 above)
Comment (Vany) [reply logicrunner/logicrunner_test.go]: It's temporaryly broken.

**Pull request merged**